erpnext/docs/user/manual/en/selling/articles/applying-discount.md

```diff
@@ -1,31 +1,25 @@
-<h1>Applying Discount</h1>
+#Applying Discount
 
-There are two ways Discount can be applied on an items in the sales transactions.
+There are several ways Discount can be applied on an items in the sales transactions.
 
 #### 1. Discount on "Price List Rate" of an item
 
-In the Item table of transaction, after Price List Rate field, you will find Discount (%) field. Discount Rate applied in this field will be applicable on the Price List Rate of an item.
+In the Item table, after Price List Rate field, you will find Discount (%) field. Discount Rate applied will be applicable on the Price List Rate of an item.
 
-Before applying Discount (%). 
+<img alt="Discount Percentage" class="screenshot" src="{{docs_base_url}}/assets/image/articles/discount-1.png">
 
-![Before discount]({{docs_base_url}}/assets/img/articles/Selection_00616c670.png)
+The feature of % Discount is available in all the sales and purchase transactions.
 
-After applying Discount (%) under Discount on Price List Rate (%) field.
+You can also %Discount applied based on **Pricing Rule** as well. [Click here to learn how Pricing Rule functions.]({{docs_base_url}}/user/manual/en/accounts/pricing-rule.html)
```

Comment:
Better version:
*You can use Pricing Rule to automatically apply Discount (%) ... how to use Pricing Rule.*